_posts/2019-05-10-meet-us-at-barcelona.md

```diff
@@ -0,0 +1,67 @@
+categories:
+  - community
+  - workshop
+  - events
+  - kubernetes
+author_staff_member: alex
+dark_background: true
+---
+
+I'll outline a few places where you can find us if you'd like to catch a session, grab a coffee, swag or just chat about Kubernetes and Serverless
+
+### Monday 20th May - Serverless Practioners Summit
+
+The [Serverless Practioners Summit](https://spseu19.sched.com) is a co-located event hosted by the [CNCF](https://www.cncf.io) at KubeCon. It's on Monday so it's technically a "day 0" event. If you planned your travel accordingly, then we'd love to see you there. The event is being organised by the OpenFaaS Founder and the companies that are behind Knative such as IBM and Google.
+
+In the morning we'll have talks and updates from the OpenFaaS and Knative projects followed by an update from the CNCF's Serverless Working Group. In the Afternoon there will be additional talks as well as open discussion sessions called Birds of a Feather or (BoF). The Google team will be running an afternoon of sponsored content on Knative.
+
+* [A bright 2019 for OpenFaaS - project & community updates](https://spseu19.sched.com/event/Nq4i/a-bright-2019-for-openfaas-project-community-updates)
+
+In the opening talk Alex will share highlights and lessons learned from the past 12 months of community development. During that time we've seen over 17k GitHub stars, 40 companies sign-up to the end-user community and over 200 contributors have PRs successfully merged.
+
+* [Kubernetes and AWS Lambda can play nicely together with OpenFaaS](https://spseu19.sched.com/event/Nq4u/kubernetes-and-aws-lambda-can-play-nicely-together-with-openfaas)
+
+Before lunch Ed Wilde will show us how innovation can come from mixing together two existing technologies to create a new one. Ed came up with the idea of faas-lambda over the holiday period last year and has since developed an interface (provider) for the OpenFaaS ecosystem that enables it to target AWS Lambda.
+
+* [OpenFaaS Birds of a Feather Session](https://spseu19.sched.com/event/Nq5C/bof-open-faas)
+
+Join Alex, contributors and other end-users for an open session where you can ask questions, see demos and find out more about the project's roadmap and future.
+
+### 2. Tuesday 22th May
+
+* [Accelerating the Journey of an AI Algorithm to Production with OpenFaaS @ BT](https://kccnceu19.sched.com/event/MPeF/accelerating-the-journey-of-an-ai-algorithm-to-production-with-openfaas-joost-noppen-bt-plc-alex-ellis-vmware)
+
+Joost Noppen from BT (British Telecom) will join OpenFaaS founder Alex Ellis to talk about how his team were able to accelerate the delivery of AI to end-users. Joost shares some of the problems his team works on and how functions on Kubernetes with OpenFaaS were able to speed-up their delivery from 3 years to just 3 months. Alex will also share some of the recent innovation in the project and updates on what to expect in 2019.
```

Comment:
I think 'British Telecom' is outdated now, they're just BT.

Joost _will_ share

Reply:
I was told that many Americans don't know what "BT" is, and therefore to put it in brackets or pronounce it fully. That was only advice I was given by a third-party, I haven't consulted BT branding guidelines.